apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/exception/Exceptions.kt

```diff
@@ -99,8 +99,33 @@ class CacheMissException(val key: String, val fieldName: String? = null) : Apoll
 class HttpCacheMissException(message: String, cause: Exception? = null) : ApolloException(message = message, cause = cause)
 
 /**
- * Multiple exceptions happened, for an example with a [CacheFirst] fetch policy
- * [cause] might change in future where both exceptions will be added as suppressed exception
+ * Multiple exceptions happened. This happens typically when a [CacheFirst] request fails to fetch
+ * both the cache and the network. In that case, [ApolloCompositeException] is thrown, and you
+ * can access the underlying cache and network exceptions in [suppressedExceptions].
+ *
+ * ```
+ * val response = try {
+ *   apolloClient.query(sampleQuery).fetchPolicy(CacheFirst).execute()
+ * } catch (e: ApolloException) {
+ *   when (e) {
+ *     is ApolloCompositeException -> {
+ *       e.suppressedExceptions.forEach { suppressed ->
+ *         handleException(suppressed)
+ *       }
+ *     }
+ *     else -> handleException(suppressed)
```

Comment:
```suggestion
 *     else -> handleException(e)
```

Reply:
Oh my! I shouldn't try to write code in comments directly...